### PR TITLE
DRAFT: WebAssembly prerendering

### PR DESCRIPTION
**Pull request by stsrki**
The Bar component still expands only after the app is fully loaded. I guess we should do a major Bar rewrite to make it work as intended.

Note: https://github.com/aspnet/AspLabs/tree/main/src/QuickGrid

### CODE REVIEW

**Comment by David-Moreira** [comment]
So is this working better now? What's different from the last time we tried it? 

Let me look up the PR.

**Comment by David-Moreira** [comment]
Hmm we didn't exchange thoughts on the PR itself, we probably discussed it over Discord, I'll have to look it up. But it's similar, so same issues might be here.

https://github.com/Megabit/Blazorise/pull/3115

File: Demos/Blazorise.Demo.Bootstrap.Host/Program.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+{
+    public class Program
+    {
+        public static void Main( string[] args )
+        {
+            var builder = WebApplication.CreateBuilder( args );
+
+            // Add services to the container.
+
+            builder.Services.AddControllersWithViews();
```
Comment (David-Moreira) [inline Demos/Blazorise.Demo.Bootstrap.Host/Program.cs:17]: Need this or copy paste?

Comment (stsrki) [reply Demos/Blazorise.Demo.Bootstrap.Host/Program.cs:17]: Probably not.

**Pull request closed without merging**